src/components/NavigationItems.js

```diff
@@ -27,6 +28,8 @@ const getHighlightedText = (text, highlight) => {
   );
 };
 
+const isExternal = (url) => url.slice(0, 4) === 'http';
```

Comment:
FYI, JavaScript has a super nice string method [`startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) that allows you to check if the beginning of the strings starts with another. This obviously works as well, but figured you'd appreciate the tip.